torch/nn/qat/modules/embedding_bag.py

```diff
@@ -0,0 +1,62 @@
+import torch
```

Comment:
Can we call this file embedding_ops.py so that in the future we can also add support for nn.Embedding modules in it?